src/System.Drawing.Common/src/System/Drawing/Bitmap.Serializable.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing
+{
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    public sealed partial class Bitmap : Image
+    {
+        /// <summary>
+        /// Constructor used in deserialization
+        /// </summary>
```

Comment:
Comments like this probably aren't necessary :)  But it's fine as you're just moving stuff around.